websites/G/Goodreads/presence.ts

```diff
@@ -0,0 +1,35 @@
+const presence = new Presence({
+	clientId: "765622778317373440"
+}),
+elapsed = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo-1"
+	},
+	path = window.location.href.replace(window.location.origin, ''),
+	urlParamshere = new URLSearchParams(window.location.search);
+
+	if(path === '/'){
+		presenceData.details = "Browsing Home";
+	}else if(path.startsWith('/book/show/')){

```

Comment:
```suggestion
	} else if (path.startsWith('/book/show/')) {
```

Spacing to help keep with a good look

Reply:
@SeMiD does it automatically

Reply:
@SeMiD will run our prettier config on it.